sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -230,6 +231,10 @@ type networkFlowManager struct {
 	sensorUpdates chan *message.ExpiringMessage
 	centralReady  concurrency.Signal
 
+	ctxMutex    sync.Mutex
+	cancelCtx   func()
```

Comment:
Out of curiosity: why not `context.CancelFunc`?

Reply:
Not sure, but I agree with the rename.